Jenkinsfile_CNP

```diff
@@ -69,6 +70,7 @@ def archiveCoreCaseDataDefinitions(env) {
       steps.archiveArtifacts artifacts: "ccd-release-config/ccd-fpl-preview-${CHANGE_ID}-toggle-off.xlsx"
     } else {
       steps.archiveArtifacts artifacts: "ccd-release-config/ccd-fpl-${env}.xlsx"
+      steps.archiveArtifacts artifacts: "ccd-release-config/ccd-fpl-${env}-shuttered.xlsx"
```

Comment:
If we wanted to dynamically be able to switch between shuttered an non shuttered then I think the only way to do that would be to `uploadCoreCaseDataDefinitions` method such that it can be controlled by some toggle from either a launch darkly (ideal) or key vault.

Once that is done we can update the else block to select the shuttered version of the defs if it the env is prod and the shutter toggle is enabled

Reply:
This would then just require that we set the toggle and then run the master pipeline for the defs to be updated